code/modules/mob/living/living.dm

```diff
@@ -501,9 +501,16 @@ default behaviour is:
 			if(affecting_mob)
 				affecting_mob.handle_grab_damage()
 
-/mob/living/Move(a, b, flag)
+/mob/living/Move(newloc, direct, flag)
 	if (buckled)
 		return
+	// Modpack edit - pixel shifting
+	if(shifting)
+		pixel_shift(direct)
+		return FALSE
+	else if(is_shifted)
+		unpixel_shift()
+	// end modpack edit
```

Comment:
This edit will conflict with upstream.
You probably want to move it to modpack as well, since those functions work only if you include it.

I used this code in my downstream to separate from core to modpack:
```
/mob/living/Move()
	. = ..()

	if(is_shifted && !buckled)
		is_shifted = FALSE
		pixel_x = default_pixel_x
		pixel_y = default_pixel_y
```

After some tweaks for you, it will look like this:
I use this code to separate from core to modpack:
```
/mob/living/Move()
	. = ..()
	
	if(shifting)
		pixel_shift(direct)
		return FALSE
	else if((is_shifted && !buckled)
		unpixel_shift()
```